sample_data/README.md

```diff
@@ -29,6 +29,26 @@ To use the sample data, at minimum 8 GB memory should be allocated to the contai
 
 ### Use Notebook image
 
+1. Use `docker start 2020_full_ch_server` to start the container.
```

Comment:
This command is used to start the data service container, this is not part of the notebook image. And the sample data has multiple datasets, we don't know the exact dataset user use. Maybe only say start the clickhouse server container is enough.